metabrainz/model/payment.py

```diff
@@ -170,9 +170,7 @@ def process_paypal_ipn(cls, form):
         if form['payment_status'] != 'Completed':
             # TODO(roman): Convert to regular `logging.info` call when such detailed logs
             # are no longer necessary to capture.
-            get_sentry_client().captureMessage("PayPal: Payment is not completed",
-                                               level=logging.INFO,
-                                               extra={"ipn_content": form})
+            capture_message("PayPal: Payment is not completed", level="info", extra={"ipn_content": form})
```

Comment:
just confirming, have you checked that this message gets reported properly?

Reply:
Yes.